tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpMessageHandlerTests.cs

```diff
@@ -26,7 +26,7 @@ public class HttpMessageHandlerTests : TracingIntegrationTest
         public HttpMessageHandlerTests(ITestOutputHelper output)
             : base("HttpMessageHandler", output)
         {
-            SetEnvironmentVariable("DD_HTTP_CLIENT_ERROR_STATUSES", "400-499, 502,-343,11-53, 500-500-200");
```

Comment:
Wondering if we should still have integration tests for the deprecated ones to validate that they do still work?

Reply:
(but if I recall this test suite is quite large)

Reply:
Yeah since the change was only to the name I hoped adding the test for asserting the precedence was enough since they check for the ranges in other tests too: https://github.com/DataDog/dd-trace-dotnet/blob/7fa9adde56eb12badda28c82929e7f14d56b8d82/tracer/test/Datadog.Trace.Tests/Configuration/TracerSettingsTests.cs#L178-L193

Since we are not deprecating the env var I can leave it there in that line for testing porpuses.